Automattic/woocommerce-services
# Release 1.25.4

## Release PR for 1.25.4
### Changes in this release
#2248 Remove Stripe Connect functionality 
#2268 Adjust documentation links.
#2262 Carrier description on dynamic carrier registration form.
#2265 Fix broken behavior on account registration page.
#2271 Remove unused method in shipping settings view.
#2275 Fix typo in readme file

This release branch is branched from latest `develop` 

### Test notes

- Test label purchase flow
- Test live rates
- Test Store on WP.com
- Check the HS Tariff Number documentation link
- Test carrier account registration forms
- Check readme file for any typos

### Changelog
* Tweak - Remove Stripe connect functionality.
* Tweak - Remove unused method in shipping settings view.
* Fix	- Breaking behavior on account registration page.
* Add - Allows registration of additional accounts.
* Tweak - Carrier description on dynamic carrier registration form.

## Build File
Built from this branch using `php woorelease.phar simulate --product_version=1.25.4 https://github.com/Automattic/woocommerce-services/tree/release/1.25.4`

[woocommerce-services.zip](https://github.com/Automattic/woocommerce-services/files/5645441/woocommerce-services.zip)
